src/ServiceControl.Config/UI/Shell/VersionCheckerHelper.cs

```diff
@@ -44,57 +45,45 @@ static async Task<List<Release>> GetVersionInformation()
             }
         }
 
-        static readonly HttpClient httpClient = new HttpClient(new HttpClientHandler
-        {
-            AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
-        })
-        {
-            DefaultRequestHeaders =
+        static readonly HttpClient httpClient =
+            new HttpClient(new HttpClientHandler
             {
-                Accept =
-                {
-                    new MediaTypeWithQualityHeaderValue("application/json"),
-                }
-            }
-        };
+                AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
+            })
+            {
+                DefaultRequestHeaders = { Accept = { new MediaTypeWithQualityHeaderValue("application/json"), } }
+            };
 
         public class Release
         {
             public Release()
             {
             }
 
-            public Release(Version current)
+            public Release(SemanticVersion current)
             {
-                Tag = current.ToString();
+                Tag = current.ToNormalizedString();
             }
 
-            [JsonProperty("tag")]
-            public string Tag { get; set; }
+            [JsonProperty("tag")] public string Tag { get; set; }
```

Comment:
Why did these change formatting? It's not really germane to this PR and is completely different than attribute usages on most every other project in the company.

Reply:
That was not intentional... maybe some auto formatting quick